.nvmrc

```diff
@@ -0,0 +1 @@
+0.10
```

Comment:
For the Web site, I don't think we are locked into Node 0.10.x; can use latest. @bijujoseph ?